app/(services)/blog-as-service/page.tsx

```
@@ -30,8 +30,28 @@ export const metadata: Metadata = {
+      url: "https://studio1hq.com",
+    },
+    serviceType: "Technical Content Writing",
+    areaServed: "Worldwide",
+    url: "https://studio1hq.com/blog-as-service",
+  };
+
   return (
     <div className="">
+      <script
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{ __html: JSON.stringify(serviceSchema) }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)